datastore/README.md

```diff
@@ -0,0 +1,24 @@
+# Google App Engine Standard Environment Datastore Sample
+
+This sample demonstrates how to deploy an App Engine Java 8 application that
+uses Cloud Datastore for storage.
+
+See the [Google App Engine standard environment documentation][ae-docs] for more
+detailed instructions.
+
+[ae-docs]: https://cloud.google.com/appengine/docs/java/
+
+## Setup
+
+1.  Update the `<application>` tag in
+    `src/main/webapp/WEB-INF/appengine-web.xml` with your project name.
+1.  Update the `<version>` tag in `src/main/webapp/WEB-INF/appengine-web.xml`
+    with your version name.
+
+## Running locally
```

Comment:
We should use the new tooling.   appengine:run & appengine:deploy  (that way we don't need to change `<application>` & `<version>` above.

That also means you should use the standard gcloud preamble in the README